### PR TITLE
Rwc2019 challenge serving drinks

### PR DESCRIPTION
**Pull request by PetervDooren**

### CODE REVIEW

File: robot_smach_states/src/robot_smach_states/utility.py [hunk 1]
```diff
@@ -310,6 +310,37 @@ def execute(self, userdata=None):
         return 'unlocked'
 
 
+class CheckBool(smach.State):
+    def __init__(self, check_designator):
```
Comment (LarsJanssenTUe) [inline robot_smach_states/src/robot_smach_states/utility.py]: Please add docstrings!

File: robot_smach_states/src/robot_smach_states/utility.py [hunk 2]
```diff
@@ -310,6 +310,37 @@ def execute(self, userdata=None):
...
+    def execute(self, userdata=None):
+        val = self._check_designator.resolve() if hasattr(self._check_designator, "resolve") else self._check_designator
+        if val:
+            return "true"
+        else:
+            return "false"
+
+
+class ToggleBool(smach.State):
+    def __init__(self, check_designator):
```
Comment (LarsJanssenTUe) [inline robot_smach_states/src/robot_smach_states/utility.py]: Please add docstrings

File: robot_smach_states/src/robot_smach_states/utility.py (hunk 1 above)
Comment (PetervDooren) [reply robot_smach_states/src/robot_smach_states/utility.py]: done

File: robot_smach_states/src/robot_smach_states/utility.py (hunk 2 above)
Comment (PetervDooren) [reply robot_smach_states/src/robot_smach_states/utility.py]: done

**Pull request merged**